### DISCUSSION TITLE
How many octerns are selected by the organization?

### DISCUSSION THREAD

**Discussion by shubh1007** [community/community#51511] [GitHub Education] [answered]
I have applied for the project defined by appwrite. I am not sure how many octerns will be selected by Appwrite. If you know about it please do share the information.

**Comment by tusharhqq** [comment]
I think 1 octern per project but to get the exact answer you should ask this question on appwrite's discord group

**Reply by shubh1007** [reply]
Ok! Thanks for the response

**Comment by ashwinexe** [comment, marked as answer]
The organization decides how many Octerns they'd like to select for the Octernships. Make sure you give your best! :)
